staging/README.md

```diff
@@ -0,0 +1,89 @@
+# Introduction
+
+Welcome to the staging tree!
+
+This tree contains experimental code that does not (yet) meet the quality
+requirements of RIOT.
+
+The idea of this tree is to provide a place to code that offers functionality,
+but has not (yet) received the polishing and refinement that is usually
+required in RIOT. staging/ is also a place where new ideas can be shared early
+with a wider audience in order to facilitate easier collaboration.
```

Comment:
in order to... -> aiming to facilitate quicker-paced collaborative coding and code maturation ?

Reply:
"ideas are shared in order to", I feel "ideas are shared aiming to..." doesnt fit here?